### ISSUE TITLE
sbgEComReceiveCmd2: wrong timeout

### ISSUE THREAD

**Issue by SelennLamson** [SBG-Systems/sbgECom#5]
In sbgEComCmdCommon.c, function sbgEComReceiveCmd2():

I guess this function calls sbgEComReceiveAnyCmd2() with a timeout of 0 because it wants to handle the timeout by itself.
However, when calling sbgEComReceiveAnyCmd2() with a timeout of 0, if the call fails (for instance with SBG_NOT_READY), the error_code will be overwritten to SBG_TIME_OUT.

When getting an SBG_TIME_OUT error, the calling function sbgEComReceiveCmd2() does not sleep during 1ms as it should and loops again immediately in a blocking way. It does so until an answer is obtained, or the actual timeout is over.

In a non-preemptive realtime software as ours, this blocks the execution of everything else until the lib gets an Ack from the device.

My temporary fix:

```C++
// sbgEComCmdCommon.c:245
	else if (errorCode == SBG_NOT_READY)
	else if (errorCode == SBG_NOT_READY || errorCode == SBG_TIME_OUT)
```

**Comment by richardbraun** [comment]
Hello,

We have seen the problem internally and will provide a fixed version soon.

Thanks.

**Comment by rsiryani** [comment]
It has been fixed in sbgECom 3.2.4011-stable:
 [SBGECOM-324] - Fix invalid time-out errors when calling sbgEComReceiveCmd2
